secret-handshake.json

```diff
@@ -0,0 +1,142 @@
+{
+    "#": [
+        "The exercise only lays out the \"commands\" method,", 
+        "but some language tracks have tested reversing the translation and",
+        "composing the two functions, as well.",
+        "Consider the sets not under \"commands\" to be optional."
+    ],
+    "commands": {
+        "description": ["Create a handshake for a number"],
+        "cases": [
+            {
+                "description": "shake for 1",
+                "input": 1,
+                "expected": ["wink"]
+            },
+            {
+                "description": "shake for 10",
+                "input": 2,
+                "expected": ["double blink"]
+            },
+            {
+                "description": "shake for 100",
+                "input": 4,
+                "expected": ["close your eyes"]
+            },
+            {
+                "description": "shake for 1000",
+                "input": 8,
+                "expected": ["jump"]
+            },
+            {
+                "description": "shake for 11",
+                "input": 3,
+                "expected": ["wink", "double blink"]
+            },
+            {
+                "description": "shake for 11 reversed",
+                "input": 19,
+                "expected": ["double blink", "wink"]
+            },
+            {
+                "description": "shake for all",
+                "input": 16,
+                "expected": ["wink", "double blink", "close your eyes", "jump"]
+            },
+            {
+                "description": "shake for all reversed",
+                "input": 31,
+                "expected": ["jump", "close your eyes", "double blink", "wink"]
+            },
+            {
+                "description": "shake for binary",
+                "input": "101",
+                "expected": ["wink", "close your eyes"]
+            },
+            {
+                "description": "shake for reversed binary",
+                "input": "10110",
+                "expected": ["close your eyes", "double blink"]
+            },
+            {
+                "description": "invalid binary",
+                "input": "121",
+                "expected": []
+            },
+            {
+                "description": "shake for 0",
+                "input": 0,
+                "expected": []
+            },
+            {
+                "description": "shake for negative",
+                "input": -1,
+                "expected": []
+            },
+            {
+                "description": "too big",
+                "input": 32,
+                "expected": []
+            },
+            {
+                "description": "invalid string",
+                "input": "piggies",
+                "expected": []
+            },
+            {
+                "description": "invalid with numbers",
+                "input": "1piggies",
+                "expected": []
+            }
+        ]
+    },
+    "decode": {
```

Comment:
For any single-action input, solutions could legitimately give one of two possible results: the reversed version or the non-reversed version, since reversing a single action will do nothing. What shall we do about this?

One idea: Allow there to be multiple expected answers in each decode case. Most of them will have only one expected answer, but the single-action inputs will have two expected answers, one with the reverse bit being 1, and another with the reverse bit being 0.

Another possibility: Just expect it to be non-reversed, but we should document that in this JSON file if we make that expectation.

We could not test it at all, but I imagine someone will ask at some point.